### PR TITLE
Lots of code clean up

### PR DESCRIPTION
**Pull request by DustinCampbell**

### CODE REVIEW

**Comment by chuckries** [comment]
:+1: Are there any specific rules on let/var/const?

**Comment by DustinCampbell** [comment]
https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#.6la3s6b0y

**Comment by chuckries** [comment]
Thanks.

File: package.json [hunk 1]
```diff
@@ -1,373 +1,388 @@
 {
```
Comment (DustinCampbell) [inline package.json]: Sorry about the changes in this file. I ran npm install --save for a particular package and it reformatted the file.

File: src/features/commands.ts [hunk 2]
```diff
@@ -186,40 +185,24 @@ function dotnetRestore(server: OmnisharpServer) {
...
-            return path.dirname(solutionPathOrFolder);
-        }
-        else {
-            return solutionPathOrFolder;
-        }
-    }).then((solutionDirectory) => {
+    getFolderPath(solutionPathOrFolder).then(folder => {
         return runInTerminal('dotnet', ['restore'], {
-            cwd: solutionPathOrFolder
+            cwd: folder
```
Comment (DustinCampbell) [inline src/features/commands.ts]: I believe this was a bug. It should have been 'solutionDirectory' before.

**Comment by DustinCampbell** [comment]
@gregg-miskelly are you ok if I merge this?

**Comment by gregg-miskelly** [comment]
I didn't review everything (let me know if you want me to) but the debugger parts look good. :+1:

**Comment by DustinCampbell** [comment]
thanks!

**Pull request merged**